### PR TITLE
style: run prettier on all files

### PR DESCRIPTION
**Pull request by alvrs**
Context in https://github.com/latticexyz/mud/pull/630#issuecomment-1509704464

### CODE REVIEW

**Review by holic — APPROVED** [review]
Did no yaml files get changed after this was enabled?

**Comment by holic** [comment]
Oh if this is for pnpm yaml, we should prob ignore that in linting and let pnpm be the owner of formatting that

**Review by dk1a — DISMISSED** [review]
It does fix my issue, but feels weird to use prettier for pnpm-lock

**Comment by alvrs** [comment]
yeah the reason for this PR was the inconsistent formatting of the `pnpm-lock.yaml` file in https://github.com/latticexyz/mud/pull/629. Generally agree that pnpm should just be responsible for formatting that file, but for some reason formatting was different on linux, and prettier yaml formatting is the same as pnpm formatting on mac. So this PR adds prettier yaml formatting to the pre-commit hook as a last line of defence.

File: package.json [hunk 1]
```diff
@@ -45,8 +45,8 @@
     "prepare": "husky install && (forge --version || pnpm foundryup)",
     "build": "pnpm recursive run build",
     "commit": "cz",
-    "prettier:check": "prettier --check 'packages/**/*.{ts,css,md,sol}'",
-    "prettier": "prettier --write 'packages/**/*.{ts,css,md,sol}'",
+    "prettier:check": "prettier --check '**/*.{ts,css,md,sol,yaml}'",
```
Comment (holic) [inline package.json:48]: should remove yaml from here too

**Pull request merged**